modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJavaCodegen.java

```diff
@@ -1989,4 +2000,16 @@ protected void addAdditionPropertiesToCodeGenModel(CodegenModel codegenModel, Sc
         }
     }
 
+    @Override
+    public void addImportsToOneOfInterface(List<Map<String, String>> imports) {
+        if (additionalProperties.containsKey(JACKSON)) {
```

Comment:
Does this mean that GSON does't work?

Reply:
Yes. There is a GSON implementation in okhttp-gson-nextgen and another java based generator. Does the spring generator supports GSON?

Reply:
I think there is a configuration option for it, but I'm not sure how widely used ti it